src/model/currency.java

```diff
@@ -0,0 +1,16 @@
+package model;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import java.sql.Date;
+
+@Data
+@AllArgsConstructor
+public class currency {
+    private String id;
+    private String name;
+    private String code;
+    private String symbol;
```

Comment:
C3. As there is a lot attributes with same types, you can use :
**private String id, name, code, symbol;**